lib/paper_trail/record_trail.rb

```diff
@@ -485,25 +497,25 @@ def without_versioning(method = nil)
         yield @record
       end
     ensure
-      @record.class.paper_trail.enable if paper_trail_was_enabled
+      PaperTrail.request.enable_model(@record.class) if paper_trail_was_enabled
     end
 
     # Temporarily overwrites the value of whodunnit and then executes the
     # provided block.
     def whodunnit(value)
```

Comment:
Why is this method here?  Shouldn't it be deprecated in favor of using `PaperTrail.request.with(whodunnit: value) do ... end`?

Reply:
Good catch. Yes, I will deprecate it.